browsers/samsunginternet_android.json

```diff
@@ -170,6 +170,12 @@
           "status": "current",
           "engine": "Blink",
           "engine_version": "79"
+        },
+        "12.1": {
+          "release_date": "2020-07-07",
+          "status": "current",
+          "engine": "Blink",
+          "engine_version": "79"
```

Comment:
Just double-checking here: 12.1 has the same engine version as 12.0? I don't know enough about how XR is actually implemented to say whether that's surprising (e.g., you backported something from 81 to 79) or routine (XR is implemented independently of the browser engine). Thanks for your help on, this @AdaRoseCannon!

Reply:
Yes same engine, as far as i know it's not back ported it's our own implementation.